File: Part 1 - Data Preprocessing/data_preprocessing_udemy.R
dataset=read.csv('Data.csv')

#Splitting the dataset into training and test set
#install.packages('caTools')

library(caTools)
set.seed(123)
split=sample.split(dataset$Purchased,SplitRatio = 0.8)
training_set=subset(dataset,split==TRUE)
test_set=subset(dataset,split==FALSE)

#Feature SCaling Excluding categorical values
# training_set[,2:3]=scale(training_set[,2:3])
# test_set[,2:3]=scale(test_set[,2:3])

File: Part 1 - Data Preprocessing/data_preprocessing_udemy.py
import numpy as np
import matplotlib.pyplot as plt
import pandas as pd
df=pd.read_csv('Data.csv')
X=df.iloc[:,:-1].values
y=df.iloc[:,-1].values

#Splitting the dataset into training and test set

from sklearn.model_selection import train_test_split
X_train,X_test,y_train,y_test=train_test_split(X,y,test_size=0.2,random_state=0)

#Feature Scaling, 

"""from sklearn.preprocessing import StandardScaler
sc_X=StandardScaler()
X_train=sc_X.fit_transform(X_train)
X_test=sc_X.transform(X_test) """

